packages/reports/tests/unit/consumers/request/sort-test.ts

```diff
@@ -132,4 +138,26 @@ module('Unit | Consumer | request sort', function (hooks) {
 
     assert.deepEqual(getSorts(request), {}, 'The sorts are removed');
   });
+
+  test('UPDATE_COLUMN_FRAGMENT_WITH_PARAMS', function (assert) {
+    const request: RequestFragment = Store.createFragment('request', {
+      table: 'network',
+      limit: null,
+      dataSource: 'bardOne',
+      requestVersion: '2.0',
+      columns: [metricWithParam],
+      filters: [],
+      sorts: [{ ...metricWithParam, direction: 'desc' }],
```

Comment:
Can we add a few more sorts here? One with the same base but different param and another with a different base metric?